projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,39 @@
+:host {
+    --t-degrees: 0;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--tui-primary);
+    border-radius: 50%;
+    box-sizing: border-box;
+    transform-style: preserve-3d;
+    width: 1rem;
+    height: 1rem;
+    border: 0.1875rem solid var(--tui-base-02);
```

Comment:
Move border to the pseudo element and make it inherit border-color.

Reply:
why?

Reply:
`z-index: 1` also doesn't work

Reply:
Right, sorry. That's why I suggested to move border to `::after` initially, so that everything is in natural stacking order.

Reply:
fixed